scripts/termux-apk

```diff
@@ -0,0 +1,50 @@
+#!/data/data/com.termux/files/usr/bin/bash
+
+install()
+{
+    apk="$(realpath $@)"
```

Comment:
This function is supposed to be called with only one argument. But, `$@` gives false sense that this function can be called with several arguments. I'd better change it to `$1`. Or, which is certainly preferable I'd add support for several args passed to `-i` and `-u` and in this function iterate over `$@`